openapi/src/test/java/io/helidon/openapi/OpenAPIConfigTest.java

```diff
@@ -1,113 +0,0 @@
-/*
```

Comment:
Randomly adding this comment here.  Generally, I am surprised we are removing all of these tests.  Were they just moved somewhere and I missed it?

Reply:
In this PR, the SE side of OpenAPI is there minimally, only as needed to underpin the MP OpenAPI support. Whereas MP uses SmallRye (and dovetails with its configuration mechanism), full OpenAPI support will use the Swagger libraries and that configuration. So the SE OpenAPI config will be changing. Some of these tests or equivalent ones adapted to Swagger will reappear.